### PR TITLE
Mysterious owl twitter bot

### LINKED ISSUE #93: Twitter bot
# Description

twitter bot to like, retweet your tweet/with specific keyword/hashtag

## Type of issue

- [X] Feature (New Script)

## Checklist:

- [X] I have read the project guidelines.
- [X] I have checked all the existing projects, before submitting a new project issue.
- [X] I have checked previous issues to avoid duplicates.
- [X] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by Mysterious-Owl**
# Description

Created the script

Fixes #93

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by Mysterious-Owl** [comment]
update the source of images after merge

File: Scripts/Bots/Twitter_bot/Twitter_bot.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	def on_success(self, data):
+		if 'text' in data:
+			a=data['text'].lower()
+			username=data['user']['screen_name']
+			id=data['id']
+			st.create_favorite(id=id)
+   			#change 'nice tweet' too your desired retweet reply
+			st.update_status(status='Nice Tweet @'+username, in_reply_to_status_id=id)
+#enter your unique keys and tokens
+api_k='*'
```
Comment (chavarera) [inline Scripts/Bots/Twitter_bot/Twitter_bot.py:13]: please enter your unique keys and tokens in config files and parse it utilize it in your code

**Review by chavarera — CHANGES_REQUESTED** [review]

File: Scripts/Bots/Twitter_bot/Twitter_bot.py (hunk 1 above)
Comment (Mysterious-Owl) [reply Scripts/Bots/Twitter_bot/Twitter_bot.py:13]: But I get a limit of around 100 requests, how can I enter my keys and tokens in public code, the user should generate it on their own.

Comment (chavarera) [reply Scripts/Bots/Twitter_bot/Twitter_bot.py:13]: just add token  code in separate file (don't add your token :smile:   )

**Review by chavarera — CHANGES_REQUESTED** [review]
enter your unique keys and tokens  template code in config file

**Comment by Mysterious-Owl** [comment]
i added a code named config.py, which has keys and tokens

File: Scripts/Bots/Twitter_bot/Twitter_bot.py (hunk 1 above)
Comment (Mysterious-Owl) [reply Scripts/Bots/Twitter_bot/Twitter_bot.py:13]: Are you trying to say a code to generate tokens for api?

**Review by chavarera — CHANGES_REQUESTED** [review]
@Mysterious-Owl use https://docs.python.org/3/library/configparser.html to write your configuration in separate file and consumes in main program using config parser

**Comment by Mysterious-Owl** [comment]
> @Mysterious-Owl use https://docs.python.org/3/library/configparser.html to write your configuration in separate file and consumes in main program using config parser

Thanks, got to know a new thing!

**Review by chavarera — APPROVED** [review]

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @Mysterious-Owl for code

**Pull request merged**